src/AppInstallerCLICore/Workflows/MSStoreInstallerHandler.cpp

```diff
@@ -249,7 +255,9 @@ namespace AppInstaller::CLI::Workflow
 
         // Policy check
         AppInstallManager installManager;
-        if (installManager.IsStoreBlockedByPolicyAsync(s_StoreClientName, s_StoreClientPublisher).get())
+        bool bypassStorePolicy = WI_IsFlagSet(context.GetFlags(), Execution::ContextFlag::BypassStorePolicy);
+
+        if (installManager.IsStoreBlockedByPolicyAsync(s_StoreClientName, s_StoreClientPublisher).get() && !bypassStorePolicy)

```

Comment:
nit:

if (!WI_IsFlagSet(context.GetFlags(), Execution::ContextFlag::BypassStorePolicy) && installManager.IsStoreBlockedByPolicyAsync())

To avoid a potential unnecessary store policy api call